be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -220,7 +225,8 @@ Status ColumnReader::init() {
 }
 
 Status ColumnReader::new_bitmap_index_iterator(BitmapIndexIterator** iterator) {
-    RETURN_IF_ERROR(_ensure_index_loaded());
+    bool use_page_cache = !config::disable_storage_page_cache && _index_meta_use_page_cache;
```

Comment:
could we add a private class member use_page_cache? Then we could just use the class member at line 229.